assets/src/util/assignment.js

```diff
@@ -232,7 +232,14 @@ const createUserSettings = (courseId, viewName, setting) => {
   return mutation
 }
 
+const assignmentStatus = {
+  GRADED: 'Graded',
```

Comment:
Since this a an object, I feel it follow the camel casing standard. it should not be All caps

Reply:
I don't care either way but should they not be all caps?
https://google.github.io/styleguide/jsguide.html#features-objects-enums

Reply:
Ok, Agreed. It's confusing to tell if this is an object or Enum. so in this scenario this a enum. and make sense to keep it like that

Reply:
I mean it's both right?  

Lets convert MyLA to Typescript real quick

Reply:
We should .... but maybe not right now.

Reply:
We can squeeze that in before Friday no probelmo 🤣